tensorflow_graphics/rendering/camera/tests/perspective_test.py

```diff
@@ -239,18 +239,22 @@ def test_intrinsics_from_matrix_to_intrinsics_random(self):
     tensor_shape = np.random.randint(1, 10, size=(tensor_size)).tolist()
     random_focal = np.random.normal(size=tensor_shape + [2])
     random_principal_point = np.random.normal(size=tensor_shape + [2])
+    random_skew_coeff = np.random.normal(size=tensor_shape + [1])
 
     matrix = perspective.matrix_from_intrinsics(random_focal,
-                                                random_principal_point)
-    focal, principal_point = perspective.intrinsics_from_matrix(matrix)
+                                                random_principal_point,
+                                                random_skew_coeff)
+    focal, principal_point, skew_coeff = perspective.intrinsics_from_matrix(matrix)
+    random_skew_coeff = np.reshape(random_skew_coeff, (1, 1))
 
     self.assertAllClose(random_focal, focal, rtol=1e-3)
     self.assertAllClose(random_principal_point, principal_point, rtol=1e-3)
+    self.assertAllClose(random_skew_coeff, skew_coeff, rtol=1e-3)
 
   @parameterized.parameters(
-      ((2,), (2,)),
-      ((2, 2), (2, 2)),
```

Comment:
This original test for instance would fail, if the batch dimension of `skew` is not coerced before `compare_batch_dimensions` is called.

Reply:
After some discussion with @sofienbouaziz , it may make the most sense to actually just remove the shape checking for this matrix_from_intrinsics. It seems like the function will work as is ok if skew=0.0, since it's reshaped to the same shape as fx, so the shape checking is causing more problems than its solving in this case, I believe.